### ISSUE TITLE
PR's review

### ISSUE THREAD

**Issue by saisumanthkumar** [codedamn/codedamn-hacktoberfest-projects#262]
The PR's isn't reviewing, While the latest PR's is being reviewed and neglecting the older one's

**Comment by awsheeshh** [comment]
yes, true I had submitted my PR 2 days ago, it still hasn't been reviewed, but new PRs are being merged..

**Comment by deepprasad56** [comment]
This is really disgusting to see that the old PR’s are getting ignored, when everyone here has done the equal amount of hardwork for the project. I request the organisers to please review our PR’s  soon and get back to us.

**Comment by Asmit2952** [comment]
Yes true, I submitted my PR twice, still hasn't been reviewed

**Comment by saisumanthkumar** [comment]
Organiser's please do respond

**Comment by Sooraj-s-98** [comment]
Second PR was my. I created 3 PR. That is not reviewed.

**Comment by Ruzzan** [comment]
Still not reviewed what a joke maintainers

**Comment by saisumanthkumar** [comment]
something is fishy
